src/devices/realsense2/realsense2Driver.cpp

```diff
@@ -705,6 +739,7 @@ bool realsense2Driver::setDepthResolution(int width, int height)
         if (m_initialized)
         {
             fallback();
+            return false;
         }
     }
```

Comment:
The function is `bool` but I don't see any `return true`. What will return the first `if`?

Reply:
The return `true` is at the end of the function :wink:

Reply:
Sorry, I missed it. Ok, fine with me